src/tracer/models.ts

```diff
@@ -0,0 +1,22 @@
+import { Event } from '../client';
```

Comment:
IMO we should move any shared interfaces to `src/types.ts` and import from there instead of importing from the API client file into the tracer client file